openshift/templates/network-security-policies/custom-metabase-nsp.yaml

```diff
@@ -0,0 +1,70 @@
+apiVersion: template.openshift.io/v1
+kind: Template
+parameters:
+- name: NAMESPACE
+- name: GROUP_NAME
+  description: the value of the 'app' label on all of the deployments for this instance
+  value: eagle-epic
+- name: API_NAME
+  description: the value of the api deployments 'role' label without the group name appended
+  value: api
+- name: DATABASE_NAME
+  description: the value of the database deployments 'role' label without the group name appended
+  value: database
+- name: METABASE_EPIC_NAME
+  description: the value of the database deployments 'role' label without the group name appended
+  value: metabase-epic
+- name: METABASE_POSTGRES_NAME
+  description: the value of the database deployments 'role' label without the group name appended
+  value: metabase-postgres
+
+objects:
+
+
+
+# Internet Communication Policies
+
+- apiVersion: security.devops.gov.bc.ca/v1alpha1
+  kind: NetworkSecurityPolicy
+  metadata:
+    name: custom-${METABASE_EPIC_NAME}-egress-internet
+    app: ${GROUP_NAME}
+  spec:
+    description: "allow metabase to talk to the internet in ${NAMESPACE}."
+    source:
+      - - $namespace=${NAMESPACE}
+      - - $role=${METABASE_EPIC_NAME}
```

Comment:
I would add the `GROUP_NAME` to role name to be consistent with the other templates

Reply:
I left it out of this one because we're not making new ones for every pr instance, I did the same for clamav

Reply:
If you think there's still value, I can add it to both

Reply:
Yeah you're correct if we don't do PR deploy metabase and clamav then it's okay to leave those out